spec/mailers/user_mailer_spec.rb

```diff
@@ -153,51 +153,7 @@
       end
     end
   end
-  
-  describe "invite request" do
```

Comment:
It might be a good idea to keep these and get rid of the ones where all the examples start with `should`